src/TextArea/styles.js

```diff
@@ -0,0 +1,64 @@
+import css from 'styled-jsx/css'
+import { colors, theme, spacers } from '../theme.js'
+
+export const styles = css`
+    .textarea {
```

Comment:
Can be moved into the next `textarea` rule.

Reply:
I don't think so, the container is a `div.textarea` and it has a `textarea` element as a child.